src/Data/LLVM/BitCode/IR/Types.hs

```diff
@@ -116,11 +116,13 @@ parseTypeBlockEntry (fromEntry -> Just r) = case recordCode r of
   8 -> label "TYPE_CODE_POINTER" $ do
     let field = parseField r
     ty <- field 0 typeRef
-    when (length (recordFields r) == 2) $ do
-      -- We do not currently store address spaces in the @llvm-pretty@ AST.
-      _space <- field 1 keep
-      return ()
-    addType (PtrTo ty)
+    space <- AddrSpace <$> if length (recordFields r) == 2
+      then
+        field 1 numeric
+      else
+        return 0
```

Comment:
Another candidate for defaultAddrSpace, alhtough the `AddrSpace` constructor will need to move to the `then`.